samples/apps/copilot-chat-app/webapp/src/libs/models/Plan.ts

```diff
@@ -0,0 +1,18 @@
+export type IPlanInput = {
+    // These have to be capitalized to match the server response
+    Key: string;
+    Value: string;
+};
+
+export type IPlanStep = {
+    skill: string;
```

Comment:
I'm curious about this definition, I'd expect things like 'name' instead of 'function'.

Reply:
Ah I see the mapping in `sk-utilities`

Reply:
Something like

```
export type IPlan = {
    skill: string;
    function: string;
    description: string;
    stepInputs: IPlanInput[];
    steps: IPlan[];
};
```

might be better.

Reply:
Sounds good -- I'll hold off on this and you can just merge your changes from your 📝 Refactor plan model and UI to support nested steps commit in after?

Reply:
I'm not sure if/when any of that work will be going to `main` so I wouldn't count on it. I've already cherry-picked your UI work (and done this suggested change to support my scenarios) and will probably stay that way for the time being.